baselines/dom.generated.d.ts

```diff
@@ -8348,7 +8631,7 @@ interface ElementEventMap {
 }
 
 /**
- * Element is the most general base class from which all objects in a Document inherit. It only has methods and properties common to all kinds of elements. More specific classes inherit from Element.
+ * **Element** is the most general base class from which all element objects (i.e.
```

Comment:
Oh no. This is tricky too without having an actual language processor library. Something for a followup.